### PR TITLE
Maui 889708 Added UG documentation for SfAutocomplete, SfComboBox, and SfMaskedEntry controls in NET MAUI.

### PR DESCRIPTION
**Pull request by HariharanChokkalingam**
Hi Team,

Added UG documentation for SfAutocomplete, SfComboBox, and SfMaskedEntry controls in .NET MAUI, Kindly review and merge the content.

Regards,
Hariharan C

### CODE REVIEW

File: MAUI/Autocomplete/No-Results-Found.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: No Result Found in in Autocomplete control | Syncfusion
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/No-Results-Found.md:3]: use -> No Results Found

File: MAUI/Autocomplete/No-Results-Found.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+description: Learn about No Result Found support in .NET MAUI Autocomplete (SfAutocomplete) control and more details.
+platform: maui
+control: SfAutoComplete
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/No-Results-Found.md:6]: use -> SfAutocomplete

File: MAUI/Autocomplete/No-Results-Found.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+documentation: ug
+---
+# No Result Found in .NET MAUI Autocomplete  (SfAutoComplete)
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/No-Results-Found.md:9]: use -> No Results Found in .NET MAUI Autocomplete (SfAutocomplete)

File: MAUI/Masked-Entry/Validation.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+description: Learn here all about Validation support in Syncfusion .NET MAUI Masked Entry (SfMaskedEntry) control and more.
+platform: maui
+control: SfMaskedEntry
+documentation: ug 
+---
+# Validation in in .NET MAUI Masked Entry (SfMaskedEntry)
+
+## Validation Mode
+
+Input validation happens based on the value of the `ValidationMode`property. The enum values of this property are:
```
Comment (Choza-rajan) [inline MAUI/Masked-Entry/Validation.md:13]: `ValidationMode`property. -> provide space

Comment (HariharanChokkalingam) [reply MAUI/Masked-Entry/Validation.md:13]: As we are using the API property here, can I include a space for that?

File: MAUI/Autocomplete/No-Results-Found.md (hunk 3 above)
Comment (HariharanChokkalingam) [reply MAUI/Autocomplete/No-Results-Found.md]: updated.

File: MAUI/Autocomplete/No-Results-Found.md (hunk 2 above)
Comment (HariharanChokkalingam) [reply MAUI/Autocomplete/No-Results-Found.md]: updated.

File: MAUI/Autocomplete/No-Results-Found.md (hunk 1 above)
Comment (HariharanChokkalingam) [reply MAUI/Autocomplete/No-Results-Found.md]: updated.

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**